libs/gltfio/src/FilamentAsset.cpp

```diff
@@ -30,6 +30,8 @@ using namespace utils;
 namespace gltfio {
 
 FFilamentAsset::~FFilamentAsset() {
+    delete mAnimator;
```

Comment:
curious why this change? if order is important, maybe a comment?

Reply:
Accidental movement, reverted.